physics/geopotential_test.cpp

```diff
@@ -549,6 +552,249 @@ TEST_F(GeopotentialTest, ThresholdComputation) {
+        massive_body_parameters,
+        rotating_body_parameters,
+        OblateBody<ICRS>::Parameters::ReadFromMessage(
+            earth_c22_s22_message.geopotential(), earth_reference_radius)));
+    geopotential_c22_s22 =
+        Geopotential<ICRS>(earths.back().get(), /*tolerance=*/0);
+  }
+  {
+    auto earth_j2_message = earth_message;
+    for (auto& row : *earth_j2_message.mutable_geopotential()->mutable_row()) {
+      for (auto& column : *row.mutable_column()) {
+        if (column.order() != 0 || row.degree() != 2) {
+          column.clear_cos();
+          column.clear_sin();
+        }
+      }
+    }
+    earths.push_back(make_not_null_unique<OblateBody<ICRS>>(
+        massive_body_parameters,
+        rotating_body_parameters,
+        OblateBody<ICRS>::Parameters::ReadFromMessage(
+            earth_j2_message.geopotential(), earth_reference_radius)));
+    geopotential_j2 = Geopotential<ICRS>(earths.back().get(), /*tolerance=*/0);
+}
+
+  Vector<double, ICRS> direction({1, 2, 5});
+  direction = direction / direction.Norm();
+
+  auto const get_acceleration =
+      [&direction](Geopotential<ICRS> const& geopotential, Length const& r) {
+        return geopotential.GeneralSphericalHarmonicsAcceleration(
+            Instant(), r * direction, r, r * r, 1 / Pow<3>(r));
+      };
+  auto const get_radial_acceleration =
+      [&direction, &get_acceleration](Geopotential<ICRS> const& geopotential,
+                                      Length const& r) {
+        Vector<double, ICRS> const down = -direction;
+        return InnerProduct(get_acceleration(geopotential, r), down);
+      };
+  auto const get_latitudinal_acceleration =
+      [&direction, &get_acceleration](Geopotential<ICRS> const& geopotential,
+                                      Length const& r) {
+        Vector<double, ICRS> const celestial_north({0, 0, 1});
+        Vector<double, ICRS> local_north =
+            celestial_north.OrthogonalizationAgainst(direction);
+        return InnerProduct(get_acceleration(geopotential, r), local_north);
+      };
+  auto const get_longitudinal_acceleration =
+      [&direction, &get_acceleration](Geopotential<ICRS> const& geopotential,
+                                      Length const& r) {
+        Vector<double, ICRS> const down = -direction;
+        Bivector<double, ICRS> const north({0, 0, 1});
+        Vector<double, ICRS> local_east = down * north;
+        return InnerProduct(get_acceleration(geopotential, r), local_east);
+      };
+
+  // Above the  outer threshold for J2.
+  EXPECT_THAT(
+      get_acceleration(earth_geopotential, 5'000'000 * Kilo(Metre)).Norm(),
+      Eq(0 / Pow<2>(Metre)));
+
+  {
+    // Inspect the J2 sigmoid.
+    Length const s0 = earth_geopotential.degree_damping()[2].inner_threshold();
+    Length const s1 = earth_geopotential.degree_damping()[2].outer_threshold();
+    EXPECT_THAT(s0, IsNear(1'500'000 * Kilo(Metre)));
```

Comment:
Check `s1` too, even if it's a fixed factor of 3 at the moment.  Same comment at other places below.